src/validators/keystores/local.py

```diff
@@ -118,29 +119,38 @@ def list_keystore_files() -> list[KeystoreFile]:
         keystores_password_file = settings.keystores_password_file
 
         res: list[KeystoreFile] = []
-        for f in listdir(keystores_dir):
-            if not (isfile(keystores_dir / f) and f.startswith('keystore') and f.endswith('.json')):
-                continue
 
-            password_file = keystores_password_dir / f.replace('.json', '.txt')
-            if not isfile(password_file):
-                password_file = keystores_password_file
+        for current_path, _, files in walk(keystores_dir):
+            for f in files:
+                file_path = Path(current_path) / f
+                if not (isfile(file_path) and f.startswith('keystore') and f.endswith('.json')):
+                    continue
 
-            password = LocalKeystore._load_keystores_password(password_file)
-            res.append(KeystoreFile(name=f, password=password, password_file=password_file))
+                # check for password file in the keystores_password_dir
+                password_file = keystores_password_dir / f.replace('.json', '.txt')
+                if not isfile(password_file):
 
+                    # check for password file in the same directory as the keystore file
+                    password_file = Path(current_path) / f.replace('.json', '.txt')
+                    if not isfile(password_file):
+
+                        # use password file from the keystores_password_file setting
+                        password_file = keystores_password_file
```

Comment:
Another nested level is not necessary
```suggestion
                if not isfile(password_file):

                    # use password file from the keystores_password_file setting
                    password_file = keystores_password_file
```